pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Summary
+
+[Change!] Describe your feature, problems you had, notes, improvements and others.
+
+## Screenshots
+
+[Change!] Show the screenshots of the views you modified.
```

Comment:
We can delete these lines

Reply:
It's true that I usually don't attach a screenshot to a PR description but currently I'm code reviewing a project that uses this a lot to show the changes that they are doing in each PR so I'm not sure of removing it.